ts/@live-compositor/web/src/manager/wasmInstance.ts

```diff
@@ -0,0 +1,101 @@
+import { ApiRequest, CompositorManager, RegisterOutputRequest } from '@live-compositor/core';
+import { Renderer, Resolution, Component, ImageSpec } from '@live-compositor/browser-render';
+import { Api } from 'live-compositor';
+
+type Output = {
+  resolution: Resolution;
+};
+
+export type OnRegisterCallback = (event: object) => void;
+
+class WasmInstance implements CompositorManager {
+  private renderer: Renderer;
+  private outputs: Map<string, Output>;
+  private onRegisterCallback: (cb: OnRegisterCallback) => void;
+
+  public constructor(props: {
+    renderer: Renderer;
+    onRegisterCallback: (cb: OnRegisterCallback) => void;
+  }) {
+    this.renderer = props.renderer;
+    this.onRegisterCallback = props.onRegisterCallback;
+    this.outputs = new Map();
+  }
+
+  public async setupInstance(): Promise<void> {}
+
+  public async sendRequest(request: ApiRequest): Promise<object> {
+    const paths = request.route.split('/');
```

Comment:
have you looked into using some route parser library? I would assume there is something standalone that is not ship with http server